chapter_introduction/index_vn.md

```diff
@@ -1993,7 +1993,13 @@ As a result, researchers have studied a number of
 *special cases* of reinforcement learning problems.
 -->
 
-*dịch đoạn phía trên*
+Vấn đề chung của học tăng cường là một thiết lập rất tổng quát.
+Các hành động này có ảnh hưởng đến những quan sát có tính tuần tự.
```

Comment:
```suggestion
Các hành động của tác nhân có ảnh hưởng đến những quan sát về sau.
```

Câu này có ý là hành động ở thời điểm hiện tại của tác nhân có ảnh hưởng tới những quan sát mà nó nhìn thấy sau này.

Sau khi dịch hết phần về RL có lẽ mình phải rà soát lại ngay toàn bộ. Văn của các bác này cứ phải đọc hết mới hiểu được.